pxr/usd/usdLux/lightAPI.h

```diff
@@ -359,10 +421,119 @@ class UsdLuxLightAPI : public UsdAPISchemaBase
     // --------------------------------------------------------------------- //
     // NORMALIZE 
     // --------------------------------------------------------------------- //
-    /// Normalizes power by the surface area of the light.
-    /// This makes it easier to independently adjust the power and shape
-    /// of the light, by causing the power to not vary with the area or
-    /// angular size of the light.
+    /// Normalizes the emission such that the power of the light
+    /// remains constant while altering the size of the light, by dividing the
+    /// luminance by the world-space surface area of the light.
+    /// 
+    /// This makes it easier to independently adjust the brightness and size
+    /// of the light, by causing the total illumination provided by a light to
+    /// not vary with the area or angular size of the light.
+    /// 
```

Comment:
this is easy with radiometric units :]

Reply:
It's just as easy with photometric units :)